src/Storage/Database/Prefill/RecordContentGenerator.php

```diff
@@ -453,9 +456,21 @@ private function getRandomTags($count = 5)
      *
      * @return array|null
      */
-    private function getRandomImage($type)
+    private function getRandomImage($type, $placeholder = null)
     {
         $pathKey = $type === 'image' ? 'file' : 'filename';
+
+        if ($placeholder) {
+            try {
+                $image = $this->apiClient->getImage($placeholder);
+                $filename = sprintf('placeholder_%s.jpg', substr(md5(microtime()), 0, 12));
+                $this->filesystem->write('files://' . $filename, $image);
+                return [$pathKey => $filename, 'title' => 'placeholder', 'alt' => 'placeholder'];
+            } catch (\Exception $e) {
```

Comment:
This is not a good idea, you don't want to catch everything silently, it makes it *extremely* hard to debug.

So …
- `getImage()`  can throw a `GuzzleHttp\Exception\RequestException` if there is a problem handling the call
- `write()` will throw a `Bolt\Filesystem\Exception\FileExistsException` that can be avoided by calling `put()`

If other exceptions are being thrown, we want to know about them as something else is fundamentally wrong.